### PR TITLE
Add a nitrocli(1) man page

### PR DESCRIPTION
**Pull request by robinkrahl**
The man page could also be moved to a subdirectory like `doc` or `etc` if you want to keep the main project directory clean.

### CODE REVIEW

File: nitrocli/nitrocli.1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.TH NITROCLI 1
+.SH NAME
+nitrocli \- access Nitrokey devices
+.SH SYNOPSIS
+.B nitrocli
+\fBclear\fR|\fBclose\fR|\fBopen\fR|\fBstatus\fR
```
Comment (d-e-s-o) [inline nitrocli/nitrocli.1]: It appears uncommon to have the subcommand appear bold. Any particular reason to do that? Or am I just looking at the wrong role model?

File: nitrocli/nitrocli.1 [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.SH DESCRIPTION
```
Comment (d-e-s-o) [inline nitrocli/nitrocli.1]: It appears all `man` pages that I've looked into include what appears to be the date of creation and the program version. E.g., for `sed` I can see:
`TH SED "1" "March 2018" "sed 4.5" "User Commands"`

Should we include that as well?

**Comment by d-e-s-o** [comment]
Yeah, I think we should put that into a `doc/` sub directory. May I ask how you created the man page? Did you just write it by hand or use a tool to convert it from something more friendly to the editor's eyes?

File: nitrocli/nitrocli.1 (hunk 1 above)
Comment (robinkrahl) [reply nitrocli/nitrocli.1]: `man-pages(7)` says:
> **SYNOPSIS**
> […]
> For  commands, this shows the syntax of the command and its arguments (including options);
> boldface is used for as-is text and italics are used to indicate replaceable arguments.

As the commands are “as-is text”, I chose boldface.

File: nitrocli/nitrocli.1 (hunk 2 above)
Comment (robinkrahl) [reply nitrocli/nitrocli.1]: We could. I left it out as I did not see a clear advantage and as it is an additional field to maintain.  If we add it, we should use the format `YYYY-MM-DD` for the date to conform to `man-pages(7)`. I’d leave out the source and manual fields as they have sensible defaults.

**Comment by robinkrahl** [comment]
Previously, I’ve used POD ([example](https://git.ireas.org/nkotp/tree/nkotp.1.pod)) to generate man pages, and I saw that there is a Rust crate for the generation ([rust-clique/man](https://github.com/rust-clique/man)).

This time, I wrote the man page directly (based on [this guide](https://liw.fi/manpages/)).  In the end, it’s not that complicated once you get used to it, and it makes it easier to deal with version control.

File: nitrocli/nitrocli.1 (hunk 2 above)
Comment (robinkrahl) [reply nitrocli/nitrocli.1]: I added the date.

**Comment by d-e-s-o** [comment]
> Previously, I’ve used POD ([example](https://git.ireas.org/nkotp/tree/nkotp.1.pod)) to generate man pages, and I saw that there is a Rust crate for the generation ([rust-clique/man](https://github.com/rust-clique/man)).
> 
> This time, I wrote the man page directly (based on [this guide](https://liw.fi/manpages/)). In the end, it’s not that complicated once you get used to it, and it makes it easier to deal with version control.

Sounds good. We can always change that at a later point in time.

**Comment by d-e-s-o** [comment]
Looks great! Thanks Robin

**Pull request merged**